apps/docs/app/features/routes/ressurser/design-tokens/OutlineTokens.tsx

```diff
@@ -1,6 +1,99 @@
-import type { BoxProps } from "@vygruppen/spor-react";
+import tokens from "@vygruppen/spor-design-tokens";
+import {
+  Box,
+  BoxProps,
+  Code,
+  Stack,
+  Table,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+} from "@vygruppen/spor-react";
 import { SharedTokenLayout } from "./SharedTokenLayout";
+import { useTokenFormatter } from "~/features/tokens/useTokenFormatter";
+
+type OutlineToken = {
+  key: "none" | "sm" | "md" | "lg" | "sm-dashed" | "md-dashed" | "lg-dashed";
+  value: string;
+};
+
+const outlineTokens: OutlineToken[] = [
+  {
+    key: "sm",
+    value: "1px solid",
+  },
+  {
+    key: "md",
+    value: "2px solid",
+  },
+  {
+    key: "lg",
+    value: "3px solid",
+  },
+  {
+    key: "sm-dashed",
+    value: "1px dash",
+  },
+  {
+    key: "md-dashed",
+    value: "2px dash",
+  },
+  {
+    key: "lg-dashed",
+    value: "3px dash",
```

Comment:
Jeg som sa feil her i dag

```suggestion
    value: "1px dashed",
  },
  {
    key: "md-dashed",
    value: "2px dashed",
  },
  {
    key: "lg-dashed",
    value: "3px dashed",
```